### PR TITLE
Update exported function details with source code comments.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@xing-yang any idea why the test fails ? somehow travis log is not verbose and locally running the failed line gives me below.
 
```
[root@localhost external-snapshotter]# go fmt $(go list ./... | grep -v vendor) | wc -l 
0
[root@localhost external-snapshotter]# 
```

File: pkg/connection/connection.go [hunk 1]
```diff
@@ -72,6 +72,7 @@ var (
 	_ CSIConnection = &csiConnection{}
 )
 
+//New returns a CSI connection object.
```
Comment (xing-yang) [inline pkg/connection/connection.go]: Add a space after "//"

File: pkg/controller/csi_handler.go [hunk 2]
```diff
@@ -42,6 +42,7 @@ type csiHandler struct {
 	snapshotNameUUIDLength int
 }
 
+// NewCSIHandler returns a handler which include the csi connection and Snapshot name details
```
Comment (xing-yang) [inline pkg/controller/csi_handler.go]: s/include/includes

File: pkg/controller/util.go [hunk 3]
```diff
@@ -68,7 +68,7 @@ const (
 )
 
 var snapshotterSecretParams = deprecatedSecretParamsMap{
-	name: "Snapshotter",
+	name:                         "Snapshotter",
```
Comment (xing-yang) [inline pkg/controller/util.go]: I think this change will cause "go fmt" to fail in 1.10.3.

File: pkg/connection/connection.go (hunk 1 above)
Comment (humblec) [reply pkg/connection/connection.go]: Done.

File: pkg/controller/csi_handler.go (hunk 2 above)
Comment (humblec) [reply pkg/controller/csi_handler.go]: Done.

File: pkg/controller/util.go (hunk 3 above)
Comment (humblec) [reply pkg/controller/util.go]: ah.. It looks like thats causing it. I have removed this change for now.

**Pull request merged**